src/components/dataset/query/QueryView.jsx

```diff
@@ -57,7 +57,7 @@ export class QueryView extends React.PureComponent {
         runQuery(
           dataset.dataProject,
           `#standardSQL
-          SELECT * EXCEPT (${DB_COLUMNS.ROW_ID}) FROM \`${dataset.dataProject}.datarepo_${dataset.name}.${selected}\`
```

Comment:
using this to assign the `key` property to each row in the table, but still hiding it from the user